app/controllers/api/v1/bathrooms_controller.rb

```diff
@@ -4,9 +4,9 @@ class BathroomsController < ApplicationController
       skip_before_action :verify_authenticity_token, only: [:create]
 
       def index
-        if params[:query]
-          bathrooms = Bathroom.where('establishment LIKE ?', "%#{params[:query].capitalize}%")
-
+        location = Geokit::Geocoders::GoogleGeocoder.geocode(params[:query])
+        if location.success
+          bathrooms = Bathroom.by_distance(:origin => location)
```

Comment:
the hash rocket `=>` is old syntax, changing it to `origin: location` should work, and not be deprecated